File: FeedleDataTier/FeedleDataTier/Network/UpdateReactionRequest.cs
using System.Text.Json.Serialization;
using FeedleDataTier.Models;

namespace FeedleDataTier.Network
{
    public class UpdateReactionRequest : Request
    {
        [JsonPropertyName("postReaction")]
        public PostReaction PostReaction { get; set; }
        public UpdateReactionRequest(PostReaction postReaction) : base(RequestType.UpdateReactionRequest)
        {
           PostReaction = postReaction;
        }
    }
}

File: Backend/src/main/java/com/feedle/feedleapi/Networking/UpdateReactionRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.PostReaction;

public class UpdateReactionRequest extends Request {
    private PostReaction postReaction;
    public UpdateReactionRequest(PostReaction postReaction)
    {
        super(RequestType.UpdateReactionRequest);
        this.postReaction = postReaction;
    }

    public PostReaction getPostReaction() {
        return postReaction;
    }
}
